external/dhcpc/dhcpc.c

```diff
@@ -163,7 +163,7 @@ struct dhcpc_state {
 	struct in_addr serverid;
 	struct in_addr ipaddr;
 	struct in_addr netmask;
-	struct in_addr dnsaddr;
+	struct in_addr dnsaddr[2];
```

Comment:
add some macro to make it understandable, dont use hardcode values

Reply:
Done